advanced_functionality/distributed_tensorflow_mask_rcnn/mask-rcnn-efs.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
<p>The AWS samples Mask R CNN actually uses TF version 1.12 (although the pip wheel says 1.13)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>